README.md

```diff
@@ -245,6 +250,50 @@ route('/page-2')  // appends a history entry
 route('/page-3', true)  // replaces the current history entry
 ```
 
+### Nested Route Example
+
+```js
+import { h, render } from 'preact'
+import Router from 'preact-router'
+import { Link } from 'preact-router/match'
+
+function Foo() {
+    return (
+      <div>
+        <h1>Foo</h1>
+        <Router>
+          <Bar path="/foo/bar" />
+          <Baz path="/foo/baz" />
+        </Router>
+      </div>
+    )
+}
+
+function Bar() {
+  return <h2>Bar</h2>
+}
+
+function Baz() {
+  return <h2>Baz</h2>
+}
+
+
+function App() {
+  return (
+    <div>
+      <Router>
+        <Foo path="/foo/:partical_path*" />
+      </Router>
+      <Link activeClassName="active" href="/foo">Foo</Link><br />
+      <Link activeClassName="active" href="/foo/bar">Bar</Link><br />
+      <Link activeClassName="active" href="/foo/baz">Baz</Link>
+    </div>
+  )
+}
+
+render(<App />, document.body)
+```
```

Comment:
```suggestion
```js
import { h, render } from 'preact';
import Router from 'preact-router';

function Profile(props) {
    // `props.rest` is the rest of the URL after "/profile/"
    return (
      <div>
        <h1>Profile</h1>
        <Router>
          <MyProfile path="/profile/me" />
          <UserProfile path="/profile/:user" />
        </Router>
      </div>
    );
}
const MyProfile = () => (<h2>My Profile</h2>);
const UserProfile = (props) => (<h2>{props.user}</h2>);

function App() {
  return (
    <div>
      <Router>
        <Home path="/" />
        <Profile path="/profile/:rest*" />
      </Router>
      <nav>
        <a href="/">Home</a>
        <a href="/profile/me">My Profile</a>
        <a href="/profile/alice">Alice's Profile</a>
      </nav>
    </div>
  );
}

render(<App />, document.body);
```
```